### ISSUE TITLE
Feature Request: Custom html title

### ISSUE THREAD

**Issue by behrs** [jdrews/logstation#30]
Please create a configuration parameter to set custom html title other than "logstation". I want to differ multiple logstation browser tabs...

And thanks for this nice tool. It's great.

**Comment by jdrews** [comment]
Good idea!

**Comment by jdrews** [comment]
I've implemented this in release [0.3.10](https://github.com/jdrews/logstation/releases/tag/0.3.10)

Here's the config for it in logstation.conf
```# Unique name for logstation instance
#    This name will be prepended to the browser tab
#    Can be useful when connecting to multiple logstations
logStationName="dc10srv42"```

![image](https://user-images.githubusercontent.com/172766/42130891-cc14e292-7cc0-11e8-8db6-5f136254172b.png)
